lib/rucio/core/account_counter.py

```diff
@@ -172,3 +156,20 @@ def update_account_counter(account, rse_id, session=None):
 
     for update in updated_account_counters:
         update.delete(flush=False, session=session)
+
+
+@transactional_session
+def update_account_counter_history(account, rse_id, session=None):
```

Comment:
I think this should be renamed to `account_usage_history`, since there are no account counters anymore. It should also be moved to `account.py`.

Reply:
In that case, we should also move `update_account_counter`. But then there is almost nothing left in `account_counter.py`. Should we move everything to `account.py` ? But similarly, why do we keep `account_limit.py` ?

Reply:
account_limit we shouldn't keep either I think. Everything account.py
The only thing where account_counter might make sense is for the methods which touch the updated_account_counter table. But even methods like add_account_counter are only used in the tests right now.

Reply:
if we can get everything into `account.py` that would be great

Reply:
OK, but the change is too big to be out for 1.19.0. Let's do it for 1.20.0

Reply:
Rebased